test/AaveAtokenFork.t.sol

```diff
@@ -0,0 +1,174 @@
+// SPDX-License-Identifier: Unlicensed
+pragma solidity >=0.8.10;
+
+import { DSTestPlus } from "solmate/test/utils/DSTestPlus.sol";
+import { Vm } from "forge-std/Vm.sol";
+import { ERC20 } from "openzeppelin-contracts/contracts/token/ERC20/ERC20.sol";
+
+import { IAToken } from "aave-v3-core/contracts/interfaces/IAToken.sol";
+import { AToken } from "aave-v3-core/contracts/protocol/tokenization/AToken.sol";
+import { IPool } from 'aave-v3-core/contracts/interfaces/IPool.sol';
+import { ConfiguratorInputTypes } from 'aave-v3-core/contracts/protocol/libraries/types/ConfiguratorInputTypes.sol';
+import { PoolConfigurator } from 'aave-v3-core/contracts/protocol/pool/PoolConfigurator.sol';
+
+import {GovToken} from "./GovToken.sol";
+
+contract AaveAtokenForkTest is DSTestPlus {
+  uint256 forkId;
+  Vm vm = Vm(address(uint160(uint256(keccak256("hevm cheat code")))));
+
+  IAToken aToken;
+  GovToken token;
+  IPool pool;
+
+  function setUp() public {
+    // We need to use optimism for Aave V3 because it's not (yet?) on mainnet.
+    // https://docs.aave.com/developers/deployed-contracts/v3-mainnet
+    uint256 optimismForkBlock = 26332308; // The optimism block number at the time this test was written.
+    forkId = vm.createSelectFork(vm.envString("OPTIMISM_RPC_URL"), optimismForkBlock);
```

Comment:
We should create a `.env.example` file with this, and add instructions for setting it up to the README